environments/data-platform.json

```diff
@@ -33,6 +33,45 @@
           "level": "read-only"
         }
       ]
+    },
+    {
+      "name": "test",
+      "access": [
+        {
+          "github_slug": "data-platform-labs",
+          "level": "view-only"
+        },
+        {
+          "github_slug": "data-platform-labs",
+          "level": "view-only"
+        }
+      ]
+    },
+    {
+      "name": "preproduction",
+      "access": [
+        {
+          "github_slug": "data-platform-labs",
+          "level": "view-only"
+        },
+        {
+          "github_slug": "data-platform-labs",
+          "level": "view-only"
```

Comment:
you have two view only's should one of them not be developer?

Reply:
Corrected